src/Surfnet/StepupMiddleware/ApiBundle/Identity/Entity/RaSecondFactor.php

```diff
@@ -98,7 +98,7 @@ class RaSecondFactor implements JsonSerializable
     public $email;
 
     /**
-     * @ORM\Column(type="stepup_document_number")
+     * @ORM\Column(type="stepup_document_number", nullable=true)
```

Comment:
That would have been nice indeed. I could still add it using some `jedi force push` magic, but I'm pretty sure that is frowned upon.